rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -7609,6 +7609,40 @@ public void getQueuedPegoutsCount_after_RSKIP271_activation() throws IOException
         assertEquals(2, bridgeSupport.getQueuedPegoutsCount());
     }
 
+    @Test
+    public void getEstimatedFeesForNextPegOutEvent_before_RSKIP271_activation() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withActivations(activations)
+                .withProvider(provider)
+                .build();
+
+        verify(provider, never()).getFeePerKb();
+        assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
+    }
+
+    @Test
+    public void getEstimatedFeesForNextPegOutEvent_after_RSKIP271_activation() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+        when(provider.getReleaseRequestQueueSize()).thenReturn(3);
+        when(provider.getFeePerKb()).thenReturn(Coin.MILLICOIN);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withProvider(provider)
+                .withActivations(activations)
+                .withBridgeConstants(bridgeConstants)
+                .build();
+
+        assertEquals(Coin.valueOf(75900), bridgeSupport.getEstimatedFeesForNextPegOutEvent());
```

Comment:
I don't like the idea of asserting against a magic value of `75900`. How was that calculated? It would be better if the test does the same calculation, taking into account release request queue size and fee per kb values

Reply:
Resolved